modules/web/js/ballerina/components/annotation-container.jsx

```diff
@@ -314,7 +314,13 @@ class AnnotationContainer extends React.Component {
             };
         });
 
-        return packageNameSuggestions.filter(packageNameObject => regex.test(packageNameObject.name));
+        packageNameSuggestions.forEach((sug) => {
+            if ((sug.name).includes(escapedValue)) {
+                matches.push(sug);
+            }
+        });
```

Comment:
We can use the filter function here right ? instead for forEach and adding it to an array.

return packageNameSuggestions.filter((sug) => { return sug.name.includes(escapedValue); }));